packages/core/integration-tests/test/integration/differing-bundle-urls/foo/one/deep.js

```diff
@@ -0,0 +1,2 @@
+sideEffect("deep");
+console.log("deep");
```

Comment:
The test shouldn't contain log statements so that the mocha output doesn't become cluttered with them

<img width="815" alt="Bildschirmfoto 2021-04-19 um 20 34 50" src="https://user-images.githubusercontent.com/4586894/115285881-b3bd9800-a14e-11eb-9657-4b655899dee6.png">